core/client/pom.xml

```diff
@@ -154,6 +154,29 @@
   </dependencies>
 
   <profiles>
+    <!-- Profile used to build alluxio client jar used for Apache Flink. This will be activated by using -Pflink -->
+    <!-- in the maven command line. -->
+    <profile>
+      <id>flink</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
```

Comment:
oh I see the spark profile is also having this ( I thought I removed this). feel free to keep this line here, we can address them later together.